content/en/blog/gdpr-territorial-scope/index.md

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Territorial scope of the GDPR",
+    "date": "2020-08-04T12:37:15+02:00",
+    "type": "blog",
+    "description": "Who is entitled to assert the GDPR's data subject rights? Against whom? An overview of the territorial scope of the GDPR.",
+    "featured_image": "gdpr-territorial-scope",
+    "tags": ["dsgvo", "räumlicher Anwendungsbereich", "gdpr", "artikel 3 dsgvo" ],
+    "authors": [ "anikarini", "baltpeter" ],
+    "license": "cc-by-40",
+    "notices": "Photo adapted after: “[aerial photography of buildings](https://unsplash.com/photos/6M9xiVgkoN0)” by [Mohit](https://unsplash.com/@98mohitkumar) ([Unsplash license](https://unsplash.com/license))"
+}
+
+Who is entitled to assert the GDPR's data subject rights? Against whom? This article provides a brief overview of the territorial scope of the GDPR.
+
+{{< img name="gdpr-territorial-scope" alt="Territorial scope of the GDPR" >}}
+
+## What is the GDPR? 
+
+The General Data Protection Regulation (Regulation (EU) 2016/679, GDPR) is a European law that came into force in 2016. The GDPR is a regulation and as such applies generally and directly to all EU member states (pursuant to Article 288 (2) TFEU). In contrast to directives, the GDPR has effects on all member states without the need for implementation into national law. It has been applicable to all EU member states since May 25, 2018.  
+
+The GDPR ensures the harmonisation and equal standards of data protection law between the member states. It also allows them to deviate from certain provisions through so-called *opening clauses*. This gives the EU member states **their own regulatory leeway** and the possibility of concretising the GDPR by adapting and modifying the provisions to their own national law.
+
+## Territorial scope of the GDPR
+
+The territorial scope of the GDPR is established in Article 3 GDPR and has not only territorial but also extraterritorial effects. Furthermore, the GDPR provides for uniform rules between companies located inside and outside Europe. The following is stipulated:
+
+### Establishment principle (paragraph 1)
+
+Here, personal data is processed by the activities of a **branch** of the controller or processor **within the Union**. For example, the GDPR applies if a company based outside Europe has a branch in Europe and processes data. An establishment is a fixed presence that carries out an effective and real activity. Thus, simply having a mailbox address or the website of a company located outside the EU being accessed is not sufficient. The legal structure of the establishment is irrelevant.
```

Comment:
I think "here" at the start of the sentence is a bit confusing, something like "in this paragraph" might be nicer to read.